thrust/thrust/detail/complex/catrig.h

```diff
@@ -671,7 +671,7 @@ _CCCL_HOST_DEVICE inline double real_part_reciprocal(double x, double y)
  * Re(catanh(z)) = x/|z|^2 + O(x/z^4)
  *    as z -> infinity, uniformly in x
  */
-#if THRUST_CPP_DIALECT >= 2011 || THRUST_HOST_COMPILER != THRUST_HOST_COMPILER_MSVC
+#if _CCCL_STD_VER >= 2011 || !defined(_CCCL_COMPILER_MSVC)
```

Comment:
Can we just drop all those? We require C++11 so there is no harm is just dropping the noise

Reply:
Dropped.